lib/src/sheet/sheet.dart

```diff
@@ -96,6 +96,30 @@ class Sheet {
     _countRowsAndColumns();
   }
 
+  /// Removes a cell from the specified [rowIndex] and [columnIndex].
+  ///
+  /// If the specified [rowIndex] or [columnIndex] does not exist,
+  /// no action is taken.
+  ///
+  /// If the removal of the cell results in an empty row, the entire row is removed.
+  ///
+  /// Parameters:
+  ///   - [rowIndex]: The index of the row from which to remove the cell.
+  ///   - [columnIndex]: The index of the column from which to remove the cell.
+  ///
+  /// Example:
+  /// ```dart
+  /// final sheet = Spreadsheet();
+  /// sheet.removeCell(1, 2);
+  /// ```
+  void _removeCell(int rowIndex, int columnIndex) {
```

Comment:
This method was introduced to simplify the process of removing cells from the sheet. Previously, when cells were deleted using row and column indices, and the deleted cell happened to be the last one in its row, the corresponding row in `_sheetData` remained as an empty entry. With this method, the row is now also deleted if its last cell is removed.

Its potential usefulness elsewhere in the codebase is not immediately clear. However, there may be other instances where cells from `_sheetData` are removed, and using this method could prove beneficial.